content/en/boilerplates/helm-prereqs.md

```diff
@@ -2,16 +2,15 @@
 ---
 ## Prerequisites
 
-1. [Download the Istio release](/docs/setup/getting-started/#download).
-
 1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
 
 1. Check the [Requirements for Pods and Services](/docs/ops/deployment/requirements/).
 
 1. [Install a Helm client](https://helm.sh/docs/intro/install/) with a version higher than 3.1.1.
```

Comment:
We should probably bump this up. From Helm's support policy

> As of Helm 3, Helm is assumed to be compatible with n-3 versions of Kubernetes it was compiled against.

and

> It is not recommended to use Helm with a version of Kubernetes that is newer than the version it was compiled against, as Helm does not make any forward compatiblility guarantees.

We should bump this up to 3.6 for Istio 1.12. There shouldn't be any issues, but this will also help in things like plugin support for kustomize, etc.